common/src/main/java/org/valkyrienskies/mod/mixinducks/feature/seamless_copy/ChunkCompileTaskDuck.kt

```diff
@@ -0,0 +1,8 @@
+package org.valkyrienskies.mod.mixinducks.feature.seamless_copy
+
+import net.minecraft.client.renderer.chunk.ChunkRenderDispatcher.RenderChunk
+
+interface ChunkCompileTaskDuck {
```

Comment:
Kotlin in the java source?! Even if it does i think its a better idea to make it java for consistency